Dockerfile

```diff
@@ -4,13 +4,18 @@ RUN apk add --no-cache git musl-dev gcc && \
   git clone https://github.com/sapcc/syslog-stdout /go/src/github.com/sapcc/syslog-stdout && \
   go build -ldflags '-s -w -linkmode external -extldflags -static' -o /usr/bin/syslog-stdout github.com/sapcc/syslog-stdout/src
 
+  git clone https://github.com/sapcc/swift-health-exporter /go/src/github.com/sapcc/swift-health-exporter && \
+  go build -ldflags '-s -w -mod=vendor' -o /usr/bin/swift-health-exporter github.com/sapcc/swift-health-exporter
```

Comment:
Does this work? I guess that you need the same ldflags as above. Otherwise this will produce a dynamic binary linking against musl libc which will not work on Debian.

Reply:
As discussed on Slack, this is not an issue.